### PR TITLE
make hoverable anchor links

### PR DESCRIPTION
**Pull request by afeld**
...using [AnchorJS](http://bryanbraun.github.io/anchorjs/).

demo

### CODE REVIEW

File: assets/js/before-you-ship.js [hunk 1]
```diff
@@ -1,9 +1,25 @@
...
+    // https://github.com/Modernizr/Modernizr/blob/da22eb27631fc4957f67607fe6042e85c0a84656/feature-detects/touchevents.js#L40
+    return ('ontouchstart' in window) || window.DocumentTouch && document instanceof DocumentTouch;
+  };
+
+  // http://bryanbraun.github.io/anchorjs/
+  if (isTouchScreen()) {
+    anchors.options = {
+      visible: 'always'
+    };
+  }
```
Comment (afeld) [inline assets/js/before-you-ship.js]: Asked about making this the default: https://github.com/bryanbraun/anchorjs/issues/47

**Comment by DavidEBest** [comment]
I like the effect. `node_modules` should be added to `.gitignore`.

**Comment by afeld** [comment]
Well, I'm not sure that [Pages](https://github.com/18F/pages-server) runs an `npm install`...do you mean that all files under `node_modules` besides the ones loaded on the page should be ignored by Git?

/cc @mbland

**Comment by mbland** [comment]
Pages definitely doesn't do that. But it _could..._ Willing to consider it, or to consider adding a generic build mechanism, like a `.18f-pages.yml` that contains something like:

```yaml
build:
- npm install
- bundle exec jekyll build
- PROFIT!
```

**Comment by DavidEBest** [comment]
My mistake. :) This good to merge, then?

**Comment by afeld** [comment]
@DavidEBest If you're good, I'm good!

**Pull request merged**